ratel/src/astgen/statement.rs

```diff
@@ -202,6 +202,17 @@ impl<'ast> SerializeInLoc for IfStatement<'ast> {
         })
     }
 }
+impl<'ast> SerializeInLoc for ImportDeclaration<'ast> {
+    fn serialize<S>(&self, serializer: S) -> Result<S::SerializeStruct, S::Error>
+    where
+        S: Serializer,
+    {
+        self.in_loc(serializer, "ImportDeclaration", 1, |state| {
+            // state.serialize_field("specifiers", &self.specifiers)?;
```

Comment:
Seems like stale code? :)

Reply:
I'm sorry. I forgot to implement astgen. I implemented it now.